delta/messages.adoc

```diff
@@ -36,6 +36,9 @@ Same format as <<{m3}.adoc#identifiers, identifiers>>; must be unique for all co
 [[clientIdType]]clientId:: Id of a <<client>>.
 Same format as <<{m3}.adoc#identifiers, identifiers>>; must be stable over time for the same client to use <<qry-GetAvailableIds, reserved free node ids>>.
 
+[[repositoryIdType]]RepositoryId:: Id of a <<repository>>.
```

Comment:
If we avoid this, we can use user facing names also as repository ids. If we enforce this we force to have a user facing name and an id